cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -149,7 +149,7 @@
 
   <property>
     <name>twill.java.reserved.memory.mb</name>
-    <value>250</value>
+    <value>${master.service.memory.mb}</value>
```

Comment:
This will break. This is the amount of Java memory to be reserved for non-heap use. Setting it to the container size means all memory will be reserved.